virtual-setup/working-with-your-own-data.md

```diff
@@ -78,6 +78,26 @@ To see the complete list of `wget` options, use the command: `wget -h` in Termin
 
 [See more `wget` examples](https://www.tecmint.com/10-wget-command-examples-in-linux/)
 
+As is recommended and also shown with this example, this dataset is zipped.
+This means after you successfully `wget` the file, you will need to unzip it.
+To unzip the contents to a particular directory, we will use the `-d` option.
+
+*Template*
+```
+unzip -d <DIRECTORY_TO_UNZIP_TO> <FILE_TO_UNZIP>
+```
+
+*Specific example:*
+Here we will unzip the contents of data/some_array_data.zip to be saved to the directory `data/`.
+First, however, we'll need to make the directory `data` using a command `mkdir`.
```

Comment:
if they downloaded it already to `data/`, they don't need to recreate it. `unzip` will also create directories, at least if they are only one level deep from currently existing directories.
```suggestion
```